src/main/java/Ryan.java

```diff
@@ -0,0 +1,308 @@
+
+    /**
+     * Commands for the different cases
+     */
+    private static final String COMMAND_BYE = "bye";
+    private static final String COMMAND_TODO = "todo";
+    private static final String COMMAND_EVENT = "event";
+    private static final String COMMAND_DEADLINE = "deadline";
+    private static final String COMMAND_DONE = "done";
+    private static final String COMMAND_LIST = "list";
+
+    /**
+     * Max number of tasks that can be stored
+     */
+    private static final int TASK_SIZE = 100;
+
+    /**
+     * Main entry point of the application and starts the interaction with user
+     */
+    public static void main(String[] args) {
+        printWelcomeMessage();
+        startChat();
+        printByeMessage();
+    }
+
+    /**
+     * Prints the goodbye message
+     */
+    private static void printByeMessage() {
+        System.out.println(MESSAGE_GOODBYE);
+    }
+
+    /**
+     * Prints the welcome message
+     */
+    private static void printWelcomeMessage() {
+        System.out.println(MESSAGE_WELCOME);
+    }
+
+    /**
+     * Prints a divider
+     */
+    private static void printDivider() {
+        System.out.println(DIVIDER);
+    }
+
+    /**
+     * Prints the list of tasks stored
+     *
+     * @param tasks array that stores the different types of tasks
+     * @param taskCounter number of tasks in the array
+     */
+    private static void printList(Task[] tasks, int taskCounter) {
+        for (int i = 0; i < taskCounter; i++) {
+            System.out.println(tasks[i]);
+        }
+        printTaskNumber(taskCounter);
+    }
+
+    /**
+     * Prints the number of tasks
+     *
+     * @param taskCounter number of tasks in the array
+     */
+    private static void printTaskNumber(int taskCounter) {
+        String task = TASK_PLURAL;
+        if (taskCounter == 1) {
+            task = TASK_SINGLE;
+        }
+        System.out.println(PRINT_TASK_MESSAGE_FRONT + taskCounter + SPACING + task + PRINT_TASK_MESSAGE_BACK);
+    }
+
+    /**
+     * Scans for the description of the input task
+     *
+     * @param chatInput input of the user
+     * @return description of task
+     */
+    private static String scanDescription(String chatInput) {
+        String[] words = chatInput.split(SPACING);
+
+        int spaceCount = 1;
+
+        int startIdx = words[0].length() + spaceCount;
+        int endIdx = 0;
+
+        for (int i = 0; i < words.length; i++) {
+            if (words[i].charAt(0) == '/') {
+                break;
+            }
+            endIdx += words[i].length() + spaceCount;
+        }
+        return chatInput.substring(startIdx, endIdx - spaceCount);
+    }
+
+    /**
+     * Scans for the event time/due date of deadline of task
+     *
+     * @param chatInput input of user
+     * @return either the due date of deadline or event time
+     */
+    private static String scanTimePeriod(String chatInput) {
```

Comment:
Perhaps the name of this method could be refactored to getTimeOfEvent as it sounds more intuitive than what it currently is.